WordPress/Classes/Utility/WPThemeSettings.m

```diff
@@ -0,0 +1,71 @@
+#import "WPThemeSettings.h"
+
+static NSString *WPThemeSettingsUserDefaultsKey = @"WPThemeSettingsUserDefaultsKey";
+static NSString *WPThemeSettingsURLScheme = @"themes";
+static NSString *WPThemeSettingsURLQueryTurnOff = @"enabled=0";
+static NSString *WPThemeSettingsURLQueryTurnOn = @"enabled=1";
+
+@implementation WPThemeSettings
+
+#pragma mark - URL handling
+
++ (BOOL)handleURL:(NSURL *)url
+{
+    BOOL result = NO;
+    NSString *query = [url query];
+    
+    if ([query isEqualToString:WPThemeSettingsURLQueryTurnOn]) {
+        [self enable];
+        result = YES;
+    } else if ([query isEqualToString:WPThemeSettingsURLQueryTurnOff]) {
```

Comment:
Are you sure about this?  I don't mind changing it that much, but it'd basically mean a URL such as the following ones would effectively turn themes OFF:

`wpdebug://themes?enable=yes`
`wpdebug://themes?enable=true`
`wpdebug://themes?enable=1&`

Let me know.